scraper.env.example

```diff
@@ -0,0 +1,7 @@
+export PGUSER=scraper-staging
+export PGPASSWORD=...
+export PGHOST=ep-....eu-central-1.aws.neon.tech
+export PGDATABASE=scraper-staging
+export SCRAPE_ENDPOINT=http://localhost:9898
+export SCRAPE_ENVIRONMENT=staging
+export SCRAPE_INTERVAL=99
```

Comment:
Add a newline. vs code is quite tricky with this, I think there was a global setting to end all files with newline as in "text file" (ends in newline) vs. "binary file" (does not end in newline).